src/main/java/com/tstepnik/planner/domain/Statistics.java

```diff
@@ -0,0 +1,72 @@
+package com.tstepnik.planner.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.net.Inet4Address;
+
+@Entity
+public class Statistics {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long userId;
+
+    private Double productivity;
+
+    private Integer numberOfTasks;
+
+    private Integer numberOfFinishTasks;
```

Comment:
This and the field right above could use better naming

Reply:
you mind missed "archived" word or i should completely rename these fields ?

Reply:
Yeah, we could use something like: `archivedTasksCount` & `finishedTasksCount`

Reply:
Remember that task can be finished but not archived :( because we only archive a task once it has expired. Do you think we should change the logic responsible for archiving tasks?

Reply:
We could but sometimes user can misclick and set task as done even it's not. Or change mind and decide that he rather didn't finish the task at all and need to change it etc.

Reply:
I think if he miss clicked he can always go to archived tab/retrieve archived tasks (which will include finished tasks as well) and change it back. At least that is how Todoist works

Reply:
Idea was he musn't do that. It's a big weight to DB if have to filtr tasks ?

Reply:
bump